src/components/ButtonComponents/NumberButtons/Numbers.js

```diff
@@ -1,19 +1,18 @@
 import React from "react";
+import NumberButton from "./NumberButton"
```

Comment:
You've imported NumberButton component but you're not using it in your return statement.